### ISSUE TITLE
How about add retweet function ?

### ISSUE THREAD

**Issue by 6p5ra** [elie222/twitter-bot#1]

**Comment by elie222** [comment]
Yup that would be good.

We need to work out a way to do this without getting blocked by Twitter. If
you overdo it your account gets blocked often.

Elie

**Comment by 6p5ra** [comment]
sleep(120) after each retweet,like,follow,reply?

**Comment by elie222** [comment]
Yeah could do something like that. At the moment it just streams all tweets
that match your search and likes it all. I assume retweeting has a similar
limit. Should be able to do a fair number per day.
